# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: Cargo.toml
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Regressor Bug: 1818349
# File Overlap Count: 24
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf Cargo.toml
--- a/Cargo.toml	Wed Mar 01 05:53:30 2023 +0000
+++ b/Cargo.toml	Wed Mar 01 17:52:01 2023 -0500
@@ -170,12 +170,12 @@
 cssparser = { git = "https://github.com/servo/rust-cssparser", rev = "b196a164dcbb317016d4aa6c58c13147e6045ebb" }
 
 # application-services overrides to make updating them all simpler.
-interrupt-support = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
-sql-support = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
-sync15 = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
-tabs = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
-viaduct = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
-webext-storage = { git = "https://github.com/mozilla/application-services", rev = "5433b3dc9ac152a0da29ed6642df839d79515a59" }
+interrupt-support = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
+sql-support = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
+sync15 = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
+tabs = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
+viaduct = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
+webext-storage = { git = "https://github.com/bendk/application-services", rev = "ecb35df5fc40357c49922f90e86bf4147fa52953" }
 
 # Patch mio 0.6 to use winapi 0.3 and miow 0.3, getting rid of winapi 0.2.
 # There is not going to be new version of mio 0.6, mio now being >= 0.7.11.